pyproject.toml

```
@@ -0,0 +1,66 @@
+[build-system]
+requires = ["setuptools>=62.6", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dask-image"
+authors = [{name="dask-image contributors. see https://github.com/dask/dask-image/graphs/contributors"}]  # noqa: E501
+description = "Distributed image processing"
+keywords = ["dask-image", "dask", "image"]
+readme = "README.rst"
+license = { file="LICENSE.txt" }
+dynamic = ["version"]
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "dask[array,dataframe] >=2023.2.0",
+    "numpy >=1.11.3",
+    "scipy >=0.19.1",
+    "pandas >=2.0.0",
+    "pims >=0.4.1",
+    "tifffile >=2018.10.18",
+]
+
+[project.optional-dependencies]
+test = [
+    "flake8 >=3.4.1",
+    "Flake8-pyproject",
+    "pytest >=3.0.5",
+    "pytest-flake8>=1.0.5",
+    "pytest-timeout >=1.0.0",
+]
+gpu = [
+    "cupy >=9.0.0",
+]
+
+[project.urls]
+"Homepage" = "https://image.dask.org"
+"Bug Tracker" = "https://github.com/dask/dask-image/issues"
```

Slight preference for "Issue" as it is more general than "Bug" (e.g. feature requests)

```suggestion
"Issue Tracker" = "https://github.com/dask/dask-image/issues"
```